prisma/migrations/20240513040814_add_settings_table/migration.sql

```diff
@@ -0,0 +1,24 @@
+-- CreateEnum
+CREATE TYPE "ChannelSyncOptions" AS ENUM ('clientAndCompany');
+
+-- CreateTable
+CREATE TABLE "Settings" (
+    "id" UUID NOT NULL DEFAULT gen_random_uuid(),
+    "workspaceId" VARCHAR(32) NOT NULL,
+    "internalUserId" UUID NOT NULL,
```

Comment:
can this be `senderId`?

Reply:
n/a since this completely changes context of this column as per our discussion